docker-compose-dev-local-images.yaml

```diff
@@ -3,24 +3,12 @@ version: '3'
 
 # Build only (put your images in here)
 services:
```

Comment:
why are the stanzas below being removed from the dev version?

Reply:
store-json, driver-os-monitor and app-os-monitor are needed for testing so have been moved into core as they are no longer optional.   Store-json will be replaced by store-timeseries when its ready.